src/gt4py/next/program_processors/runners/dace/transformations/redundant_array_removers.py

```diff
@@ -97,6 +107,8 @@ class CopyChainRemover(dace_transformation.SingleStateTransformation):
     node_a1 = dace_transformation.PatternNode(dace_nodes.AccessNode)
     node_a2 = dace_transformation.PatternNode(dace_nodes.AccessNode)
 
+    _direction: str
```

Comment:
```suggestion
    direction = dace_properties.Property(dtype=str, ...)
```

However, if you follow with my proposal to remove it as an _argument_, then you no longer need it.